config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -222,14 +222,14 @@ periodics:
       - --provider=gce
       - --gcp-zone=us-west1-b
       - --gcp-node-image=gci
-      - --extract=ci/latest-1.29
+      - --extract=ci/latest-1.30
       - --extract-ci-bucket=k8s-release-dev
       - --timeout=150m
       - --test_args=--ginkgo.focus=\[Slow\] --ginkgo.skip=\[Driver:.gcepd\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
       - --ginkgo-parallel=30
       env:
       image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240311-b09cdeb92c-master
-      resources: &id002
```

Comment:
tests are failing because this was deleted. it's basically declared here and used in 3 other places 

https://cs.k8s.io/?q=id002&i=nope&files=&excludeFiles=&repos=